src/components/flipper.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from '@/styles/Flipper.module.scss'
+import Image from 'next/image'
+import { useState } from 'react';
+import { FlipperImages } from '@/data/ice-age-texas/flipper/flipperData'
```

Comment:
@DanMyers300 we need the specific `ice-age-texas` data load to happen inside the `pages/stories/ice-age-texas` and then be passed to the `flipper` component please.  You can see if you

Reply:
Okay so I can kinda conceptualize how this is going to work but I don't have the technical skills to pull it off. Need to do some more research into React props and how all of that works

Reply:
Done